api/v1alpha1/clusterextension_types.go

```diff
@@ -46,27 +46,66 @@ const (
 
 // ClusterExtensionSpec defines the desired state of ClusterExtension
 type ClusterExtensionSpec struct {
-	// source is a required field which selects the installation source of content
+	// resolution is a required field which selects the installation source of content
```

Comment:
We had a discussion about this in the RFC and it looks like I had forgotten to update appropriately in the issue that this PR is linked to, but we want this to remain named `source` instead of `resolution`. This applies for all the types  associated with this as well.

My apologies for dropping the ball on updating the issue!